source/guide_seafile.rst

```diff
@@ -0,0 +1,271 @@
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 8,11,14,19,24
+
+  -------------------------------------------------------
+  Please choose a way to initialize seafile databases:
+  -------------------------------------------------------
+
+  [1] Create new ccnet/seafile/seahub databases
+  [2] Use existing ccnet/seafile/seahub databases
+
+  [ 1 or 2 ] 2
+
+  Which mysql user to use for seafile?
+  [ mysql user for seafile ] isabell
+
+  Enter the existing database name for ccnet:
+  [ ccnet database ] isabell_ccnet-db
+
+  verifying user "isabell" access to database isabell_ccnet-db ...  done
+
+  Enter the existing database name for seafile:
+  [ seafile database ] isabell_seafile-db
+
+  verifying user "isabell" access to database isabell_seafile-db ...  done
+
+  Enter the existing database name for seahub:
+  [ seahub database ] isabell_seahub-db
+
+  verifying user "isabell" access to database isabell_seahub-db ...  done
+
+Configure port
+--------------
+
+Since seafile uses its own webserver and fileserver, you need to find 2 free ports. Run this command 2 times and write down the 2 ports.
+
+.. include:: includes/generate-port.rst
+
+Step 5
+------
+
+Enter your domain name in config; Edit ``~/seafile/conf/ccnet.conf``
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  SERVICE_URL = https://isabell.uber.space/
+
+Step 6
+------
+
+Change seahub port in config; Edit ``~/seafile/conf/gunicorn.conf``
+
+.. warning:: Replace ``<yourport>`` with your first port!
+
+.. code-block:: console
+  :emphasize-lines: 1
+
+  bind = "0.0.0.0:<yourport>"
+
+Step 7
+------
+
+Change seafile port in config; Edit ``~/seafile/conf/seafile.conf``
+
+.. warning:: Replace ``<yoursecondport>`` with your second port!
+
+.. code-block:: console
+  :emphasize-lines: 2
+
+  [fileserver]
+  port = <yoursecondport>
+
+Step 8
+------
+
+Change seahub config; Edit ``~/seafile/conf/seahub_settings.py``
+Add the following lines:
+
+.. warning:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+  :emphasize-lines: 1,2,6,8
+
+  SITE_BASE = 'https://isabell.uber.space'
+  SITE_NAME = 'isabell.uber.space'
+
+  SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+  FILE_SERVER_ROOT = 'https://isabell.uber.space/seafhttp'
+
+  CSRF_TRUSTED_ORIGINS = ['isabell.uber.space']
+
+Step 9 - Setup .htaccess
+------------------------
+
+Create a ``~/html/.htaccess`` file with the following content:
+
+.. warning:: Replace ``<yourport>`` with your first port!
+.. warning:: Replace ``<yoursecondport>`` with your second port!
+
+.. code-block:: apache
+  :emphasize-lines: 6,7
+
+  RewriteEngine On
+  RewriteCond %{REQUEST_FILENAME} !-f
+  DirectoryIndex disabled
+  RewriteBase /
+
+  RewriteRule ^seafhttp/(.*) http://localhost:<yoursecondport>/$1 [P]
+  RewriteRule ^(.*)$ http://localhost:<yourport>/$1 [P]
+
+
+In our example this would be:
```

Comment:
the ports 9001 and 9000 aren't mentioned anywhere in the port selection above :sweat_smile: